src/doc/src/reference/environment-variables.md

```diff
@@ -231,7 +231,7 @@ corresponding environment variable is set to the empty string, `""`.
 * `CARGO_PKG_RUST_VERSION` — The Rust version from the manifest of your package.
   Note that this is the minimum Rust version supported by the package, not the
   current Rust version.
-* `CARGO_CRATE_NAME` — The name of the crate that is currently being compiled.
+* `CARGO_CRATE_NAME` — The name of the crate that is currently being compiled. It is the name of the [Cargo target] the `-` converted to `_`, such as the name of the library, binary, example, integration test, or benchmark.
```

Comment:
Just one more thing!

```suggestion
* `CARGO_CRATE_NAME` — The name of the crate that is currently being compiled. It is the name of the [Cargo target] with `-` converted to `_`, such as the name of the library, binary, example, integration test, or benchmark.
```

Reply:
Oops, we are back in the weird state 😆. Should be 

* ✅ with `-` converted to `_`
* ❌ with **the** `-` converted to `_`